CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Feat - Add Latent Batch partitions count panel to sparkIngestionndLag [Asana] (https://app.asana.com/0/1133479128121896/1202716334041153/f)
```

Comment:
```suggestion
* Feat - Add Latent Batch partitions count panel to sparkIngestionndLag [Asana](https://app.asana.com/0/1133479128121896/1202716334041153/f)
```

Reply:
The markdown format for links doesn't have a space between the text and url portions